### DISCUSSION TITLE
Red addresses? Please explain...

### DISCUSSION THREAD

**Discussion by u07** [x64dbg/x64dbg#3367] [Q&A] [answered]
Hi! It's not in the manual, so I'm a bit confused with the color scheme. Here is a jump table, and it's all red. And a strange underline or hyphen symbol is near them. 

For about a day I was sure they are relocations/fixups, but it's strange, cause the addresses are relative to RIP and so don't require relocation. So, why red and what's the hyphen for?

![image](https://github.com/x64dbg/x64dbg/assets/20968145/cebd02c0-57e1-4fde-a94b-63f202cd1d97)

**Comment by mrexodia** [comment, marked as answer]
- The red address indicates that there is a label on that address (you can see it if you make the address column bigger). If there is no comment, the label is also displayed on the right hand side (in red to match).
- The red `-` indicates that the destination of the jump is outside of the current region (eg to another module).
- Relocations would be underlined in the bytes column (which indeed is not the case here)

Hope that answers your question 🙂 (and thanks for wording it nicely and including a screenshot, made it quick and easy to answer).

**Reply by u07** [reply]
Heartfelt gratitude,  mrexodia! 😊✨✨✨✨✨
